src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -108,20 +116,20 @@ impl<'b> Resolver<'b> {
                 let module_path: Vec<Name> = match view_path.node {
                     ViewPathSimple(_, ref full_path) => {
                         full_path.segments
-                                 .split_last()
-                                 .unwrap()
-                                 .1
-                                 .iter()
-                                 .map(|seg| seg.identifier.name)
-                                 .collect()
+                            .split_last()
+                            .unwrap()
+                            .1
+                            .iter()
+                            .map(|seg| seg.identifier.name)
+                            .collect()
```

Comment:
I actually don't mind the block indentation here. Then again, I almost always prefer block indenting to visual indenting since visual indenting causes rightward drift, bad diffs, and (imo) ugliness.

That being said, I'd probably format this
```rust
full_path.segments.split_last().unwrap().1.iter().map(|seg| {
    seg.identifier.name
}).collect()
```